sim/core/spell.go

```diff
@@ -196,25 +196,11 @@ func (unit *Unit) RegisterSpell(config SpellConfig) *Spell {
 		panic(fmt.Sprintf("Over 200 registered spells when registering %s! There is probably a spell being registered every iteration.", config.ActionID))
 	}
 
-	if config.BaseDamageMultiplierAdditivePct == 0 {
-		config.BaseDamageMultiplierAdditivePct = 100
-	}
-
 	// Default the other damage multiplier to 1 if only one or the other is set.
-	if config.DamageMultiplier != 0 && config.DamageMultiplierAdditivePct == 0 {
-		config.DamageMultiplierAdditivePct = 100
-	} else if config.DamageMultiplierAdditivePct != 0 && config.DamageMultiplier == 0 {
+	if config.DamageMultiplierAdditivePct != 0 && config.DamageMultiplier == 0 {
 		config.DamageMultiplier = 1
 	}
 
-	if config.ImpactDamageMultiplierAdditivePct == 0 {
-		config.ImpactDamageMultiplierAdditivePct = 100
-	}
-
-	if config.PeriodicDamageMultiplierAdditivePct == 0 {
-		config.PeriodicDamageMultiplierAdditivePct = 100
-	}
```

Comment:
The actually useful stuff. Converting these to base 0 instead of base 100. Might want to adjust the naming though